setup.py

```diff
@@ -10,7 +10,9 @@
 
 INSTALL_DEPS = ['kubernetes==6.1.0',
                 'requests>=2.20.0',
-                'urllib3==1.22']
+                'urllib3==1.22',
+                'boto3==1.14.28',
+                'moto==1.3.14']
```

Comment:
Is moto a test-only dependency? Then shouldn't it be in TEST_DEPS?

Reply:
Moved it to test_deps